paasta_itests/environment.py

```diff
@@ -11,13 +11,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import asyncio
 import os
 import shutil
 import time
 
 import mock
 import requests
-from behave_pytest.hook import install_pytest_asserts
```

Comment:
pytest-asyncio needed an upgrade of pytest to >=3.x; pytest 3.x does assertion rewriting differently than 2.x, which breaks behave-pytest: https://github.com/ribozz/behave-pytest/issues/2